.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+| Q | A
+| --- | ---
+| Bug? | no
+| Critical? | no
+| Version Used | 1.1.x
```

Comment:
Important info, especially from first-timers. Should be updated with new minor/major releases ;)

Not sure if issues from not released code should be written as e.g. 1.2.x-dev or the exact `branch#commit-sha`?